ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellFileAccessoryViewDemoController.swift

```diff
@@ -519,8 +519,9 @@ class TableViewCellFileAccessoryViewDemoController: DemoTableViewController {
 
         for accessoryView in topAccessoryViews + bottomAccessoryViews {
             if let cell = accessoryView.tableViewCell {
-                cell.paddingLeading = TableViewCell.defaultPaddingLeading + extraPadding
-                cell.paddingTrailing = TableViewCell.defaultPaddingTrailing + extraPadding
+                let tokens = TableViewCellTokens()
+                cell.paddingLeading = tokens.paddingLeading + extraPadding
```

Comment:
If we're not using the property mentioned in one of the comments below, shouldn't we just remove it?

`@objc public var defaultPaddingLeading: CGFloat { tokens.paddingLeading }`